app/controllers/cafe_shifts_controller.rb

```diff
@@ -0,0 +1,20 @@
+# encoding:UTF-8
+class CafeShiftsController < ApplicationController
+  load_permissions_and_authorize_resource
+
+  def show
+    @cafe_shift = CafeShift.find(params[:id])
+    @cafe_shift.cafe_worker || @cafe_shift.build_cafe_worker
+  end
+
+  def feed
+    render json: CafeQueries.between(params[:start], params[:end])
+    .as_json(user: current_user)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.